internal/security/pkiinit/cert/x509Config_test.go

```diff
@@ -0,0 +1,128 @@
+//
```

Comment:
Perhaps incorporate this file into https://github.com/edgexfoundry/edgex-go/blob/master/internal/pkg/config/x509_test.go

Reply:
Ok, we'll see how we can incorporate these tests.

Reply:
@tsconn23  yes, I have incorporated the tests into the one under pkg/config/x509_test.go and pushed the changes.  Please review it again.  Thanks for your feedback!